subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/AbstractTestDescriptor.java

```diff
@@ -50,12 +48,6 @@ public TestDescriptorInternal getParent() {
         return null;
     }
 
-    @Nullable
-    @Override
-    public Object getOwnerBuildOperationId() {
```

Comment:
Did you check with the test distribution team if this breaks them?

Reply:
Ah, the whole method is marked as such...

Reply:
OK, checked with @marcphilipp. None of the closed source stuff depends on calling the method. However, the Test Distribution plugin extends `DefaultTestSuiteDescriptor` and it does implement `getOwnerBuildOperationId()`.

We'll keep the method there (sans `@Override`) to keep it working with older Gradle versions.